clitable.py

```diff
@@ -314,7 +314,7 @@ def _ParseCmdItem(self, cmd_input, template_file=None):
   def _PreParse(self, key, value):
     """Executed against each field of each row read from index table."""
     if key == 'Command':
-      return re.sub(r'(\[\[.+?\]\])', self._Completion, value)
+      return regex.sub(r'(\[\[.+?\]\])', self._Completion, value)
```

Comment:
would it be preferable to either manually disable regex's more advanced matching features manually (IE: throw an error/warning if used) because they are not supported by re or would it make more sense to make a subclass that dynamically switches between using re and regex depending on whether the user has the regex module installed? 

Personally I think the former option is a bit cleaner but the later option is backwards compatible while allowing the user some additional useful functionality.